models/questions.py

```diff
@@ -15,10 +15,10 @@
 
 # In SQL we can manually add the constraint
 # alter table questions add constraint name_bc_unique UNIQUE(name, base_course);
-try:
-    db.executesql('''alter table questions add constraint name_bc_unique UNIQUE(name, base_course)''')
-except:
-    pass
+# try:
+#     db.executesql('''alter table questions add constraint name_bc_unique UNIQUE(name, base_course)''')
+# except:
+#     pass
```

Comment:
According to everything I could find in the web2py docs, you cannot create a constraint like this as part of the regular syntax for defining a table.